src/gas-price/gas-price.ts

```diff
@@ -125,10 +106,10 @@ export const getRecommendedGasPrice = (chainId: string, providerName: string, sp
     return null;
   }
 
-  const lastUpdateTimestamp = sponsorLastUpdateTimestamp[sponsorWalletAddress];
-  // Check if the next update is a retry of a pending transaction
-  if (lastUpdateTimestamp) {
-    const pendingPeriod = Math.floor(Date.now() / 1000) - lastUpdateTimestamp;
+  // Check if the next update is a retry of a pending transaction and scale the gas price accordingly.
+  let gasPriceToUse = multiplyBigNumber(latestGasPrice, recommendedGasPriceMultiplier);
```

Comment:
It's more clear to have the calculation that multiplies by `recommendedGasPriceMultiplier` be done here 👍🏼